src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -76,6 +76,9 @@
             <li aura:id="courseConTab" id="courseConTab" class="slds-tabs__item slds-text-heading--label" title="Course Connections" role="tab">
                 <ui:outputURL aura:id="courseConTabBtn" value="#" label="{!$Label.c.stgTabCourseConn}" click="{!c.tabNavigationClick}" class="courseConTab course-con-menu-item" />
             </li>
+            <li aura:id="programPlanTab" id="programPlanTab" class="slds-tabs__item slds-text-heading--label" title="Program Plan" role="tab">
```

Comment:
@xinnanlisf @lifewithryan I don't think we need to edit this now (as our other tabs are the same way), but I wonder if we should switch to using labels for these tab titles in a future tech debt WI. I expect this is perhaps used by screenreaders and such (I might be wrong though)

Reply:
@robertbwatson I like that idea. We should create a separate WI under the Tech Debt Theme for this.

Reply:
We might have an item already. Let me see if I can find it. fyi @aligeez